lib/src/specs/class.dart

```diff
@@ -60,6 +69,23 @@ abstract class Class extends Object
       visitor.visitClass(this, context);
 }
 
+enum ClassModifier {
+  base,
+  final$,
```

Comment:
@munificent - I'm curious how you would name something referring to the `final` keyword.

This would be the first place we use a trailing `$` to avoid an otherwise unusable word. We have used `clazz` in place of `class` in a few variables and an argument, but that is mostly not visible in user code.

Reply:
This was modeled after `FieldModifier` but I would also be curious to hear @munificent's take!

https://github.com/dart-lang/code_builder/blob/e8c01b00a7327743cc01deaaba0e23346eced78d/lib/src/specs/field.dart#L61

Reply:
Ah, I forgot about that one and I was failing at writing a regex to look for it. Given the precedent we should stick with this name.

Reply:
Follow whatever convention makes sense. Personally, I tend to follow Analyzer and would do `finalKeyword`. It's a little verbose, but I like that more than arbitrarily mangling the name (`clas`, `clazz`, etc.).